src/chat/GroupList.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { View, Text, ListView, StyleSheet } from 'react-native';
+import { Avatar, Touchable, Screen } from '../common';
+import styles from '../common/styles';
+
+const moreStyles = StyleSheet.create({
+  heading: {
+    fontSize: 24,
+    marginTop: 16,
+  },
+  button: {
+    marginTop: 8,
+    padding: 16,
+  },
+  userList: {
+    marginTop: 16,
+  },
+});
+
+export default class GroupList extends React.PureComponent {
+
+  constructor(props) {
+    super(props);
+    const { recipients } = this.props;
+    const ds = new ListView.DataSource({ rowHasChanged: (r1, r2) => r1 !== r2 });
+    this.state = {
+      dataSource: ds.cloneWithRows(recipients),
```

Comment:
Move all this code in the render() function.
State is for values that the component needs to change during its lifetime.